platform/viewer/public/theme/theme.json5

```diff
@@ -0,0 +1,80 @@
+{
+  ThemeProtocols: {
```

Comment:
It is possible to define additional hanging protocols for different uses at a site by defining more themes and using theme=MoreHangingProtocols